localstack/services/s3/v3/provider.py

```diff
@@ -288,14 +289,13 @@
 SSE_ALGORITHMS = get_class_attrs_from_spec_class(ServerSideEncryption)
 OBJECT_OWNERSHIPS = get_class_attrs_from_spec_class(ObjectOwnership)
 
-# TODO: pre-signed URLS -> REMAP parameters from querystring to headers???
-#  create a handler which handle pre-signed and remap before parsing the request!
+DEFAULT_S3_TMP_DIR = "/tmp/localstack-s3-storage"
 
 
 class S3Provider(S3Api, ServiceLifecycleHook):
     def __init__(self) -> None:
         super().__init__()
-        self._storage_backend = EphemeralS3ObjectStore()
+        self._storage_backend = EphemeralS3ObjectStore(DEFAULT_S3_TMP_DIR)
```

Comment:
i mentioned this before: maybe we could allow injecting a storage backend into the constructor